### PR TITLE
TYDE: 485 - Do not return locked node content until node is actually unlocked

### PR DESCRIPTION
**Pull request by narendrasss**
This PR includes a TYDE-only change that doesn't exist in #493:

* Content for locked nodes are also sent if the user is a copilot and the node is a teen-only node

### CODE REVIEW

File: .php_cs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
..."switch_case_space":true,"visibility_required":true,"encoding":true,"full_opening_tag":true},"hashes":{"docker\/wp-config.php":1446420810,"classes\/class.tapestry-audio.php":772895317,"classes\/class.tapestry-node.php":3664630745,"classes\/class.tapestry-h5p.php":286287400,"classes\/class.tapestry-user-progress.php":3887176623,"classes\/class.tapestry-form.php":189113522,"classes\/class.tapestry.php":2917867219,"classes\/class.tapestry-group.php":357512215,"endpoints.php":44321958,"tapestry.php":119386274,"utilities\/class.tapestry-helpers.php":1381338964,"utilities\/class.tapestry-user-roles.php":1307383750,"utilities\/class.tapestry-node-permissions.php":2001037632,"utilities\/class.tapestry-errors.php":521017367,"utilities\/class.tapestry-permissions.php":3086820100,"templates\/single-tapestry.php":854946963,"interfaces\/interface.tapestry-form.php":3140436763,"interfaces\/interface.tapestry-node.php":3293252685,"interfaces\/interface.tapestry-group.php":1664109186,"interfaces\/interface.tapestry.php":1846250292,"interfaces\/interface.tapestry-user-progress.php":4176572162,"interfaces\/interface.tapestry-h5p.php":3451862674,"interfaces\/interface.tapestry-audio.php":1320567166}}
```
Comment (wynnset) [inline .php_cs.cache:1]: I have a feeling this should not be getting committed @narendrasss .

**Comment by wynnset** [comment]
We should merge this PR in ASAP as it's needed to close Sprint 27.

**Comment by narendrasss** [comment]
@wynnset Done and ready.

**Comment by wynnset** [comment]
Sorry I merged the other `tyde` PR and this has another merge conflict again 😅

**Pull request merged**